config_hub/pretrain/tinystories.yaml

```diff
@@ -1,50 +1,95 @@
-model_name: null
+
+# The name of the model to pretrain. Choose from names in ``litgpt.config``. Mutually exclusive with
+# ``model_config``. (type: Optional[str], default: null)
+model_name:
+
+# A ``litgpt.Config`` object to define the model architecture. Mutually exclusive with
+# ``model_config``. (type: Optional[Config], default: null)
 model_config:
-  name: stories15M
```

Comment:
why does it delete this?

Reply:
You need to give it the previous config so that the default values in the signature are overwritten. For example:

`litgpt pretrain --config config_hub/pretrain/tinystories.yaml --print_config=comments`

Reply:
That's what I did